libs/auth/src/certs.ts

```diff
@@ -18,7 +18,7 @@ const VX_CUSTOM_CERT_FIELD = {
   COMPONENT: `${VX_IANA_ENTERPRISE_OID}.1`,
   /** Format: {state-2-letter-abbreviation}.{county-or-town} (e.g. ms.warren or ca.los-angeles) */
   JURISDICTION: `${VX_IANA_ENTERPRISE_OID}.2`,
-  /** One of: system-administrator, election-manager, poll-worker */
+  /** One of: system-administrator, election-manager, poll-worker, poll-worker-with-pin */
   CARD_TYPE: `${VX_IANA_ENTERPRISE_OID}.3`,
```

Comment:
I decided to use the existing cert card type field as opposed to a new field, once again to avoid creating the false sense that having a PIN or not was an option for sys admins and election managers

Reply:
I like it, fewer ways to make mistakes.